sdcm/cluster.py

```diff
@@ -2812,20 +2812,20 @@ class FlakyRetryPolicy(RetryPolicy):
     A retry policy that retries 5 times
     """
 
-    def _retry_message(self, msg, *args, **kwargs):  # pylint: disable=unused-argument,arguments-differ
+    def _retry_message(self, msg, /, *args, **kwargs):  # pylint: disable=unused-argument,arguments-differ
```

Comment:
Why is there a comma after the message variable?

Reply:
https://docs.python.org/3/whatsnew/3.8.html#positional-only-parameters